packages/executor/src/utils/etherscan.ts

```diff
@@ -1,20 +1,16 @@
 import assert from 'assert'
 
-import { Contract } from 'ethers'
+import { Contract, ethers } from 'ethers'
```

Comment:
It's redundant to import `Contract` and `ethers` here, since `Contract` exists as a member on the `ethers` object. (The same applies for basically everything else in the `ethers` package)